packages/cli/README.md

```diff
@@ -781,7 +781,7 @@ OPTIONS
   --indent=indent        specify indentation for formatting JSON or YAML output
 
 EXAMPLES
-  $ smartthings capabilities:translations:upsert custom1.outputModulation 1 -i en.yml 
+  $ smartthings capabilities:translations:upsert custom1.outputModulation 1 -i en.yml
```

Comment:
Not sure where these whitespace changes are occurring. I don't see them in git locally.

Reply:
I see the same thing but I see them locally and undo them...the oclif README generation includes the spaces and my VSCode settings remove them so whenever I edit a README, I always regenerate the README to undo the removal VSCode does.

Reply:
[done](https://github.com/SmartThingsCommunity/smartthings-cli/compare/1050d875d6ac11f3fa03c20ed6c15d0096902d3d..2bda34e7d2c2f60bac6bc75702f68a776f9d3b19)